sbt-dumpschema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/model/Codecs.scala

```diff
@@ -0,0 +1,10 @@
+package org.virtuslab.akkasaferserializer.model
+
+import io.bullet.borer.Codec
+import io.bullet.borer.derivation.MapBasedCodecs.{deriveAllCodecs, deriveCodec}
```

Comment:
Hmmm Borer codecs are used within sbt-dumpschema-plugin? 😯 or is it just a temporary setup?

Reply:
Oh, is it needed for the sake of writing JSONs? 🤔

Reply:
But AFAICS you're using `scala.util.parsing.json` elsewhere...

Reply:
I am using Borer [here](https://github.com/VirtusLab/akka-safer-serializer/blob/2df15d552e1f7c4554c8d722d6a5caaeaecbc334/sbt-dumpschema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/SchemaWriter.scala#L20) and [here](https://github.com/VirtusLab/akka-safer-serializer/blob/2df15d552e1f7c4554c8d722d6a5caaeaecbc334/sbt-dumpschema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/SchemaWriter.scala#L40)